doc/source/reference/arrays.datetime.rst

```diff
@@ -20,9 +20,9 @@ Basic Datetimes
 ===============
 
 The most basic way to create datetimes is from strings in
-ISO 8601 date or datetime format. The unit for internal storage
-is automatically selected from the form of the string, and can
-be either a :ref:`date unit <arrays.dtypes.dateunits>` or a
+ISO 8601 date, datetime format or from integer to UNIX time. 
```

Comment:
I think the insertion here makes it a little confusing, because the next sentence is again about strings. How about instead adding a separate paragraph saying:

_It is also possible to create datetimes from integer. Those mean "offset relative to the Unix epoch (which is  00:00:00 UTC on 1 January 1970)"._

Reply:
thanks for feedback, I revised the text :)